app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :events
```

Comment:
It was to be next inline to add. I thought its my resposnibility to include this now. Same goes for testing of this association, I wrote it as pending but changed to reflect this.